src/org/labkey/test/WebDriverWrapper.java

```diff
@@ -3549,6 +3552,50 @@ public void actionPaste(WebElement input, String text)
         }
     }
 
+    public String getClipboardContent() throws IOException, UnsupportedFlavorException
+    {
+        DataFlavor[] flavors = Toolkit.getDefaultToolkit().getSystemClipboard().getAvailableDataFlavors();
+        Transferable t = Toolkit.getDefaultToolkit().getSystemClipboard().getContents(null);
+
+        // Adding debug info for TeamCity run.
+        // Windows is not giving DataFlavor (MIME Type) for the data on the clipboard.
+        log("Available flavors: " + Arrays.stream(flavors).toList());
+        log("Best flavor: " + DataFlavor.selectBestTextFlavor(flavors));
+
+        if (t != null)
+        {
+
+            // Adding debug info for TeamCity run.
+            log("Is DataFlavor.imageFlavor supported? " + t.isDataFlavorSupported(DataFlavor.imageFlavor));
+            log("Is DataFlavor.allHtmlFlavor supported? " + t.isDataFlavorSupported(DataFlavor.allHtmlFlavor));
+            log("Is DataFlavor.fragmentHtmlFlavor supported? " + t.isDataFlavorSupported(DataFlavor.fragmentHtmlFlavor));
+            log("Is DataFlavor.selectionHtmlFlavor supported? " + t.isDataFlavorSupported(DataFlavor.selectionHtmlFlavor));
+            log("Is DataFlavor.javaFileListFlavor supported? " + t.isDataFlavorSupported(DataFlavor.javaFileListFlavor));
+            log("Is DataFlavor.stringFlavor supported? " + t.isDataFlavorSupported(DataFlavor.stringFlavor));
+
+            DataFlavor[] transferFlavors = t.getTransferDataFlavors();
+            log("Transferable supported data flavors: " + Arrays.stream(transferFlavors).toList());
```

Comment:
You sure you want to leave this code in place? I think I added this for debugging clipboard issues on Windows and I'm not sure it is adding any value.